src/main/java/interface_adapter/generate_outfit/GenerateOutfitPresenter.java

```diff
@@ -0,0 +1,34 @@
+package interface_adapter.generate_outfit;
+
+import interface_adapter.ViewManagerModel;
+import use_case.generate_outfit.OutfitOutputData;
+import use_case.generate_outfit.OutputBoundary;
+
+public class GenerateOutfitPresenter implements OutputBoundary {
+    private final ViewManagerModel viewManagerModel;
+
+    public GenerateOutfitPresenter(ViewManagerModel viewManagerModel, GenerateOutfitViewModel generateOutfitViewModel) {
+        this.viewManagerModel = viewManagerModel;
+        this.generateOutfitViewModel = generateOutfitViewModel;
+    }
+
+    private final GenerateOutfitViewModel generateOutfitViewModel;
+
+    @Override
+    public void prepareSuccessView(OutfitOutputData outfit) {
+        GenerateOutfitState generateOutfitState = generateOutfitViewModel.getState();
+        generateOutfitState.setOutfit(outfit.outfit());
+        this.generateOutfitViewModel.setState(generateOutfitState);
+        generateOutfitViewModel.firePropertyChanged();
+
+        viewManagerModel.setActiveView(generateOutfitViewModel.getViewName());
```

Comment:
After successfully generating an outfit, this presenter updates the associated ViewModel with the generated outfit and changes the state to 'GeneratedOutfit'. However, currently, the 'GeneratedOutfit' state is still residing in the `generateOutfitViewModel`. Consider refactoring to move the 'GeneratedOutfit' state to a more appropriate location.